api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/hazelcast/HazelcastClusterProperties.java

```diff
@@ -90,6 +90,11 @@
     @RequiredProperty
     private int port = 5701;
 
+    /**
+     * The outbount ports for the Hazelcast configuration.
```

Comment:
*Outbound*.

Also, please document the property to explain meaning and intention both here, and possibly in the docs. (Though in the docs, you can just reference the setting; Javadoc is fine)

Reply:
you're right